### PR TITLE
[CORE-442] [CORE-752] Add companies.list: filter on tags

### PR DESCRIPTION
**Pull request by dietrichm**
Add docs for the added filter on tag in companies.list.

❗️ only merge when [CORE-752] is merged.

### CODE REVIEW

**Review by WouterSioen — APPROVED** [review]
👍 But let's wait to merge 'till the implementation is deployed.

**Review by JordyDeGroeve — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -611,6 +611,7 @@ Get a list of companies.
                 + email: `info@piedpiper.eu` (string, required)
             + ids: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5`,`f8a57a6f-dd1e-41a3-b8d3-428663f1d09e` (array[string], optional)
             + term: `Acme` (string, optional) - Filters on name, vat number, emails and telephones
+            + tag: `expo` (string, optional) - Filters on tag name
```
Comment (jenssegers) [inline apiary.apib]: Can we change this to `tags`? So that we can search on multiple tags at the same time? Kinda makes more sense to me :)

**Review by jenssegers — CHANGES_REQUESTED** [review]

File: apiary.apib (hunk 1 above)
Comment (dietrichm) [reply apiary.apib]: We will make a follow up PR in core to handle this.
The docs are updated.

 :ok: by @WouterSioen on 22/02/2018 11:05:18

**Review by WouterSioen — APPROVED** [review]

**Comment by WouterSioen** [comment]
@jenssegers can this be unblocked and merged? This is released

**Review by jenssegers — APPROVED** [review]

**Pull request merged**